Specs/Options/English/BooleanModel.json

```diff
@@ -60,7 +60,7 @@
 ,
 {
   "Input": "yes... I said no onions",
-  "NotSupportedByDesign": "python",
+  "NotSupportedByDesign": "python, dotNet",
```

Comment:
Why is this the case? Shouldn't it behave like the javascript version?

Reply:
Removed "dotnet not supported"

Reply:
Skip test removed